File: TransCoder_tokenized_test_set_functions/cpp/GENERATE_PYTHAGORAS_TRIPLET_SINGLE_INTEGER.cpp
void evaluate ( long long int n ) {
  if ( n == 1 || n == 2 ) printf ( "No Pythagoras Triplet exists" );
  else if ( n % 2 == 0 ) {
    long long int var = 1LL * n * n / 4;
    printf ( "Pythagoras Triplets exist i.e. " );
    printf ( "%lld %lld %lld", n, var - 1, var + 1 );
  }
  else if ( n % 2 != 0 ) {
    long long int var = 1LL * n * n + 1;
    printf ( "Pythagoras Triplets exist i.e. " );
    printf ( "%lld %lld %lld", n, var / 2 - 1, var / 2 );
  }
}

File: TransCoder_tokenized_test_set_functions/java/GENERATE_PYTHAGORAS_TRIPLET_SINGLE_INTEGER.java
static void evaluate ( int n ) {
  if ( n == 1 || n == 2 ) System . out . println ( "No Pythagoras " + "Triplet exists" );
  else if ( n % 2 == 0 ) {
    int var = 1 * n * n / 4;
    System . out . print ( "Pythagoras Triplets " + "exist i.e. " );
    System . out . print ( n + " " );
    System . out . print ( var - 1 + " " );
    System . out . println ( var + 1 + " " );
  }
  else if ( n % 2 != 0 ) {
    int var = 1 * n * n + 1;
    System . out . print ( "Pythagoras Triplets " + "exist i.e. " );
    System . out . print ( n + " " );
    System . out . print ( var / 2 - 1 + " " );
    System . out . println ( var / 2 + " " );
  }
}